src/cmake-tools.ts

```diff
@@ -909,6 +963,32 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
    */
   async clean(): Promise<number> { return this.build('clean'); }
 
+  /**
+   * Implementation of `cmake.openConfiguration`
+   */
+  async openConfiguration(): Promise<number> {
+    if (!this._webview) {
+      const drv = await this.getCMakeDriverInstance();
+      if (!drv) {
+        vscode.window.showErrorMessage('No CMakeCache.txt file has been found. Please configure project first!');
+        return new Promise(resolve => resolve(1));
```

Comment:
Do not return a Promise from an `async` function.  `async` does this auto-magically.  Just `return 1;` here.